tests/e2e-playwright/helpers/api.ts

```diff
@@ -111,31 +121,26 @@ const addLessonContent = async ( api, { id, content = lessonSimple(), ...lesson
 	} );
 };
 
-export const createCourseCategory = async ( context, category ) => {
+type Category = {
+	id?: number
+	name: string;
+	description: string;
+	slug: string
+};
+
+export const createCourseCategory = async (context: APIRequestContext, category: Category): Promise<Category> => {
 	const { name, description, slug } = category;
-	const api = await createApiContext( context );
-	return api.post( `/wp-json/wp/v2/course-category`, {
+	const api = await createApiContext(context);
+	return api.post(`/wp-json/wp/v2/course-category`, {
 		name,
-		description: description || `Some description for ${ name }`,
+		description: description || `Some description for ${name}`,
 		slug,
-	} );
+	});
 };
 
-const getNonce = async ( context ) => {
+const getNonce = async ( context: APIRequestContext ) => {
 	const response = await context.get( 'http://localhost:8889/wp-admin/admin-ajax.php?action=rest-nonce', {
 		failOnStatusCode: true,
 	} );
 	return response.text();
 };
-
-export const disableGlobalLearningMode = async ( context: BrowserContext ) => {
```

Comment:
During the course completion tests we were toggling the learning mode AND creating a course via API using the _course_theme meta. 

Both approaches lead to the same result, so I remove it and only keep the API version.

You can see the API approach [here](https://github.com/Automattic/sensei/pull/6285/files#diff-406b98a0870ba0f686807a4b817585ba0beb5ae21c32419fe09b63bf8313989bL23-L27)